airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeInternalStagingSqlOperations.java

```diff
@@ -50,7 +50,6 @@ public SnowflakeInternalStagingSqlOperations(final NamingConventionTransformer n
     this.nameTransformer = nameTransformer;
   }
 
-  @Override
```

Comment:
shouldn't we keep this annotation?

Reply:
This was removed from base interface and resurrected again. I'll add it back